framework/src/Volo.Abp.Ddd.Application.Contracts/Volo/Abp/Application/Services/IReadOnlyAppService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+using Volo.Abp.Application.Dtos;
+
+namespace Volo.Abp.Application.Services
+{
+    public interface IReadOnlyAppService<TEntityDto, in TKey>
+        : IReadOnlyAppService<TEntityDto, TEntityDto, TKey, PagedAndSortedResultRequestDto>
+    {
+
+    }
+
+    public interface IReadOnlyAppService<TEntityDto, in TKey, in TGetListInput>
+        : IReadOnlyAppService<TEntityDto, TEntityDto, TKey, TGetListInput>
+    {
+
+    }
+
+    public interface IReadOnlyAppService<TGetOutputDto, TGetListOutputDto, in TKey, in TGetListInput>
```

Comment:
Why the `IReadOnlyAppService` interface does not implement `IApplicationService`